plugins/CoreHome/angularjs/reporting-menu/reportingmenu.controller.js

```diff
@@ -219,6 +223,29 @@
             enterSubcategory(found.category, found.subcategory, found.subsubcategory);
         });
 
+        function showOnlyRawDataMessage() {
+            notification.remove(notificationId);
+
+            menuModel.fetchVisits(decodeURIComponent(date), period, function(json) {
+                if (json.value > 0) {
+                    return;
+                }
+
+                menuModel.fetchLiveVisits(decodeURIComponent(date), period, function(json) {
+                    if (json.length == 0) {
+                        return;
+                    }
+
+                    var url = piwik.broadcast.getHashFromUrl();
+                    url = piwik.broadcast.updateParamValue('category=General_Visitors', url);
+                    url = piwik.broadcast.updateParamValue('subcategory=Live_VisitorLog', url);
+
+                    var message = _pk_translate('CoreHome_PeriodHasOnlyRawData', ['<a href="' + url + '">', '</a>']);
+                    notification.show(message, {context: 'info', id: notificationId});
+                });
+            });
+        }
+
```

Comment:
@flamisz I'm still needing to think about it more but overall thinking as soon as we notice there's visits for the given segment/date then we wouldn't need to execute this request anymore on subsequent page changes maybe and could use some "cached result"? At least the live query to the get last visits wouldn't need to be executed anymore since once we know it had data once it will also have it for all subsequent page views (unless date or segment changes). 

Overall wondering if we'd maybe also want to execute this say max once every minute or so (when page is completely reloaded then it would also be executed again directly) and maybe we'd execute it say max 5 times per page view or something. Let's maybe brainstorm tomorrow